### PR TITLE
Added wimax (closes #21)

### LINKED ISSUE #21: Add WiMax as an underlying connection technology?
Android supports wimax and it would be nice to provide a max download speed for it from the spec.

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by igrigorik** [comment]
FWIW, I wouldn't put it under cellular, it's a 802.16 standard.. if anything it's closer to WiFi than cellular bucket. I'd treat it as a separate family, on par with WiFi - i.e. {cellular: [], wifi: [], wimax: [...]}

**Comment by marcoscaceres** [comment]
hmm... this means also adding to the enum. If we know that this tech is on the way out, should we really bother?

**Comment by marcoscaceres** [comment]
(made the update to the JSON file, btw)

**Comment by igrigorik** [comment]
> "In reference to economies of scale, GSM-HSPA subscribers number in the billions. Even over the next five years, the number of WiMAX subscribers is likely to be quite low. Infonetics Research projects 132 million subscribers by 2016." - [source](http://www.4gamericas.org/index.cfm?fuseaction=page&sectionid=259)

[This report](http://www.statista.com/statistics/277656/number-of-wimax-subscriber-worldwide-since-2010/) points to a lower subscriber number, but we're still talking tens of millions... And, at least, based on these sources, growing, if its at a very small rate (comparatively speaking).

---

I'd +1 adding it to the enum.

**Comment by marcoscaceres** [comment]
This also shows a lot of usage world wide: http://en.wikipedia.org/wiki/List_of_WiMAX_networks

**Comment by marcoscaceres** [comment]
Oki, added. Thanks @igrigorik for the additional data points.

File: index.html [hunk 1]
```diff
@@ -131,6 +131,8 @@
   <dd>The user agent will not contact the network when the user follows links or when a script requests a remote page (or knows that such an attempt would fail) - i.e., equivalent to `navigator.onLine === false` in HTML.</dd>
   <dt><dfn id="idl-def-ConnectionType.wifi">wifi</dfn></dt>
   <dd>The user agent is using a Wi-Fi connection as the <a>underlying connection technology</a>.</dd>
+  <dt>wimax</dt>
+  <dd>The user agent is using a WiMAX connection as the underlying connection technology.</dd>
```
Comment (igrigorik) [inline index.html:135]: For sake consistency wrap "underlying connection type" with `<a>`'s?

**Comment by igrigorik** [comment]
lgtm, thanks for making the update!

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html:135]: argh, well spotted

**Pull request merged**